File: Congestion_Collection.py
import json, requests,pymongo

r = requests.get('https://traffic.longdo.com/api/json/traffic/index?callback=get')

def get(r):
	data = r.json
	save_to_mongo(data, mongo_db, mongo_db_coll, **mongo_conn_kw)
	
def save_to_mongo(data, mongo_db, mongo_db_coll, **mongo_conn_kw):

	# Connects to the MongoDB server running on
	# localhost:27017 by default

	client = pymongo.MongoClient(**mongo_conn_kw)

	# Get a reference to a particular database

	db = client[mongo_db]

	# Reference a particular collection in the database

	coll = db[mongo_db_coll]

	# Perform a bulk insert and return the IDs

	coll.insert(data)

File: congestion_collection.js
var MongoClient = require('mongodb').MongoClient;
var url = "mongodb://localhost:27017/actual_raod";	
var requestURL ="https://traffic.longdo.com/api/json/traffic/index?callback=callback_function";
var request = new JSONHttpRequest();
request.open('GET', requestURL);
request.responseType = 'json';
request.send();

function callback_function(){

	var MongoClient = require('mongodb').MongoClient;
	var url = "mongodb://localhost:27017/actual_raod";
	
	MongoClient.connect(url, function(err, db) {
  if (err) throw err;
  var output = request.response;
  db.collection("congestion").insertOne(output, function(err, res) {
    if (err) throw err;
    console.log("1 document inserted");
    db.close();
  });
});
	
}